### DISCUSSION TITLE
Better integration with Home Assistant

### DISCUSSION THREAD

**Discussion by dhoeben** [music-assistant/support#2406] [Feature requests and ideas]
I love the Music Assistant, finally something user friendly to use on my dashboard. 

However, when using MA as a webpage view in HA, there are two topbars. Both with different fonts. 

Is there anyway to combine these better? Like hiding of the MA topbar or placing it at the bottom? Or follow HA theme?

cheers!

**Comment by OzGav** [comment]
If you access the MA UI via ingress there are no other bars?

<img width="810" alt="image" src="https://github.com/music-assistant/support/assets/19848947/50047e1b-dc97-4633-b9f8-d036eca3ef06">

**Reply by dhoeben** [reply]
@OzGav, hes I have. My navigation in HA relies on the back button. When using it as a panel, it does not provide one. This is why I said using the webpage view.

**Reply by OzGav** [reply]
That is a webpage to me. Do you mean by exposing port 8095 and then using a card?

**Reply by dhoeben** [reply]
You don't have a back button in the topbar. My view is just a yaml panel view, using a single iframe card.

**Reply by OzGav** [reply]
Is this not a duplicate of this discussion? https://github.com/orgs/music-assistant/discussions/954
